tasks/build-binary-new-cflinuxfs4/builder.rb

```diff
@@ -532,6 +532,123 @@ def build_yarn
     merge_out_data(old_filepath, filename_prefix)
   end
 
+  def build_nginx
+
+    nginx_options = [
+      '--with-cc-opt=-fPIC -pie',
+      '--with-ld-opt=-fPIC -pie -z now',
+      '--with-compat',
+      '--with-stream=dynamic',
+      '--with-http_sub_module',
+    ]
+
+    build_nginx_helper(nginx_options)
```

Comment:
Do you think it's better to avoid following the pattern build_X for this helper function, as builder.rb already sets a convention that `build_` is a prefix for first class dependency building functions?

Reply:
At the time of having to do the function, I thought of the same thing. I wanted to place it outside, maybe in a helper but semantically it should be in Build since that is what the method is doing. So if you have any suggestions on where we could move that to follow the pattern.